### PR TITLE
Fix Unzip

### PR DESCRIPTION
**Pull request by arsandev**
I have a small solution for ``unzip()`` that wasn't working before.

### CODE REVIEW

File: test.ts [hunk 1]
```diff
@@ -64,14 +64,7 @@ Deno.test("dir", async () => {
...
-  const z = await readZip("example.zip");
-  await z.unzip(dir);
-
-  const content = await Deno.readFile(join(dir, "Hello.txt"));
-  assertEquals("Hello World\n", decode(content));
-
-  const smile = await Deno.readFile(join(dir, "images", "smile.gif"));
-  assertEquals("", decode(smile));
+  const z = new JSZip();
+  await z.unzip("example.zip");
```
Comment (hayd) [inline test.ts:69]: this test no longer does anything?

Comment (arsandev) [reply test.ts:69]: just meant to make it clean and focus on its function

Comment (hayd) [reply test.ts:69]: The problem is this test no longer tests anything.

**Comment by hayd** [comment]
Can you provide an example directory structure which fails before? (that could be used as a test case).

Is it due to this issue:

```ts
         // hopefully the directory is prior to any files inside it!
```
?

File: test.ts (hunk 1 above)
Comment (hayd) [reply test.ts:69]: Previously it:

- created a temp dir
- unzipped example.zip into that dir
- asserted the presence of two files (one nested)

**Comment by hayd** [comment]
I don't think that ext = .zip is a requirement for zip files, e.g. .epub.

Closing, but thanks for taking the time for this fix (apologies for the long delay).

**Pull request closed without merging**